Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1170,8 +1176,74 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run:
                                                      '%Y-%m-%dT%H:%M:%S.000Z')
         latest_alert_id = alert_id
 
-    return incidents, \
-        {'last_fetched_alert_create_time': latest_alert_create_date, 'last_fetched_alert_id': latest_alert_id}
+    res = {'last_fetched_alert_create_time': latest_alert_create_date, 'last_fetched_alert_id': latest_alert_id}
+    return incidents, res
+
+
+def process_search_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Gets arguments for a process search task, and returns the task's id and status.
+    """
+    process_name = args.get('process_name', '')
+    process_hash = args.get('process_hash', '')
+    event_id = args.get('event_id', '')
+    query = args.get('query', '')
+    limit = args.get('limit')
+    if not limit:
+        limit = 20
+    try:
+        limit = int(limit)
+    except ValueError:
+        raise ValueError("Please provide a number as limit.")
+
+    raw_respond = client.create_search_process_request(process_name=process_name, process_hash=process_hash,
+                                                       event_id=event_id, query=query, limit=limit)
+    readable_output = f"job_id is {raw_respond.get('job_id')}."
+    output = {'job_id': raw_respond.get('job_id'), 'status': 'In Progress'}
+    return CommandResults(outputs_prefix='CarbonBlackEEDR.SearchProcess', raw_response=raw_respond,
+                          outputs=output, outputs_key_field='job_id', readable_output=readable_output)
+
+
+def event_by_process_search_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Gets arguments for an event-by-process search task, and returns the task's results.
+    """
+    process_guid = args.get('process_guid', '')
+    event_type = args.get('event_type', '')
+    query = args.get('query', '')
+    limit = args.get('limit')
+    if not limit:
+        limit = 20
```

Comment:
```suggestion
    limit = args.get('limit', '20')
```

Reply:
It fails the playbook as the test puts limit="" which exists but not empty so we get "" instead of 20.